data/mods/obsolete-mods.json

```diff
@@ -1 +1 @@
-[ "ags", "Medieval_pack", "RemoteVehicles", "RemoteControls_pack", "old_guns", "no_magazines" ]
+[ "ags", "Medieval_pack", "RemoteVehicles", "RemoteControls_pack", "old_guns", "no_magazines", "No_Diamond_Weapons" ]
```

Comment:
I'm nor sure diamond weapons are actually OP given the degree of difficulty in creating them. 

In any case you could simply avoid the CVD machine. That said if you think it's essential I'll add a special case for the CVD spawn?